contract/contracts/ERC20KavaWrappedNativeCoin.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This is the primary file that needs review.

I've extended the generic ERC20 to be Ownable and exposed the following methods:
* customizable decimals on deploy -> requires overriding decimals() view
* mint() exposed for the contract owner which will be the evmutil module
* burn() exposed for the contract owner which will be the evmutil module